README.md

```diff
@@ -56,15 +52,14 @@ into which the project was cloned and run `stack build`.
 The [`stack` system](https://docs.haskellstack.org/en/stable/README/) is
 required before `cardano-sl` can be built.
 
-Running the nodes
------------------
-
-After the project is built, run it:
-
-- open *new terminal window*, run `tmux` in it (`sudo apt-get install tmux` or else according to your distro)
-- open *new terminal window*, run `export WALLET_TEST=1; ./scripts/launch.sh` in it
```

Comment:
Maybe at least put few lines here like: `If you want to launch nodes by yourself locally see [this part](link to doc) of documentation`?